src/main/java/it/pagopa/selfcare/pagopa/config/OpenApiConfig.java

```diff
@@ -24,38 +24,39 @@
 @Configuration
 public class OpenApiConfig {
 
-    public static final String BASE_PATH = "/backoffice/external/v1";
+    public static final String BASE_PATH_PSP = "/backoffice/external/psp/v1";
+    public static final String BASE_PATH_EC = "/backoffice/external/ec/v1";
     public static final String BASE_PATH_HELPDESK = "/backoffice/helpdesk/v1";
+    public static final String LOCAL_PATH = "http://localhost:8080";
+    public static final String APIM_DEV = "https://api.dev.platform.pagopa.it";
+    public static final String APIM_UAT = "https://api.uat.platform.pagopa.it";
+    public static final String APIM_PROD = "https://api.platform.pagopa.it";
 
     @Bean
     OpenAPI customOpenAPI(
             @Value("${info.application.name}") String appName,
             @Value("${info.application.description}") String appDescription,
             @Value("${info.application.version}") String appVersion) {
         return new OpenAPI()
-                .servers(List.of(new Server().url("http://localhost:8080"),
-                        new Server().url("https://{host}{basePath}")
+                .servers(List.of(new Server().url(LOCAL_PATH),
+                        new Server().url("{host}{basePath}")
                                 .variables(new ServerVariables()
                                         .addServerVariable("host",
-                                                new ServerVariable()._enum(List.of("api.dev.platform.pagopa.it", "api.uat.platform.pagopa.it", "api.platform.pagopa.it"))
-                                                        ._default("api.dev.platform.pagopa.it"))
-                                        .addServerVariable("basePath", new ServerVariable()._enum(List.of(BASE_PATH, BASE_PATH_HELPDESK))._default(BASE_PATH))
+                                                new ServerVariable()._enum(List.of(APIM_DEV, APIM_UAT, APIM_PROD))
+                                                        ._default(APIM_DEV))
+                                        .addServerVariable("basePath", new ServerVariable()._enum(List.of(BASE_PATH_PSP, BASE_PATH_EC, BASE_PATH_HELPDESK))._default(LOCAL_PATH))
```

Comment:
Shouldn't it be BASE_PATH instead of LOCAL_PATH in the default value?

Reply:
You're right. In this case i'll remove the meaningless default